app/src/main/res/values/strings.xml

```diff
@@ -110,6 +110,15 @@
     <string name="tourbook_import_error">Fehler beim Import des Tourenbuchs</string>
     <string name="tourbook_export_error">Fehler beim Export des Tourenbuchs</string>
 
+    <!-- Ascend edit -->
+    <string name="ascend_edit_note">Bemerkung:</string>
+    <string name="ascend_edit_partner">Seilpartner:</string>
+    <string name="ascend_edit_date">Datum:</string>
+    <string name="ascend_edit_style">Stil:</string>
+    <string name="ascend_edit_title">Begehung bearbeiten</string>
+    <string name="ascend_edit_ok">OK</string>
```

Comment:
"OK" and "Abbrechen" are common names. 

- [ ] Please name the according resources to "ok" and "cancel" and move them the dialog strings in this file.